web/views/teacher.py

```diff
@@ -298,12 +300,38 @@ def build_score_csv(assignments, filename):
         response['Content-Disposition'] = f'attachment; filename="{unidecode(filename)}"'
         return response
 
+def build_edison_task_score_csv(student_points: Dict, filename: str):
+    """
+    Build CSV file importable by Edison system.
+    Edison requires delimiter set to `;`.
+    """
+    with io.StringIO() as out:
+        for student, points in student_points.items():
+            w = csv.writer(out, delimiter=';')
```

Comment:
```suggestion
        w = csv.writer(out, delimiter=';')
        for student, points in student_points.items():
```
This should probably be created just once?

Reply:
Thank you. I didn't catch that. And it surprises me a bit that it works.